admin/init.sls

```diff
@@ -0,0 +1,5 @@
+admin-packages:
+  pkg.installed:
+    - pkgs:
+      - mosh
```

Comment:
I'll put sqlite3 just on the master machine.

I guess I can add screen, but tmux is just so nice... (plus, cowsay approved!):

```
 __________ 
< tmooooox >
 ---------- 
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```

(It's 4 AM here.)